googleMapsApiModule.py

```diff
@@ -2,15 +2,17 @@
 import webbrowser
 import config
 import mysql.connector
+from constants import *
+
+gmaps = googlemaps.Client(config.key)
+#global variable gmaps
 
 def direction(origin,destination):
-    gmaps = googlemaps.Client(config.key)
     result = gmaps.directions(origin,destination)
     address = "origin="+origin+"&"+"destination="+destination
     address = address.lower()
     address = address.replace(" ","+")
-    url = "https://www.google.com/maps/dir/?api=1&"
-    result_url = url+address
+    result_url = "{}/{}".format(BASE_URL['direction'], address.lower().replace(" ", "+"))
```

Comment:
Ensure if `result_url` is formed properly. In this case, I think you should have removed the `&` at the end of `BASE_URL['direction'] and changed this line to:

```python3
result_url = "{}&{}".format(BASE_URL['direction'], address.lower().replace(" ", "+"))
```

The end-goal is to form a `result_url` like `https://www.google.com/maps/dir/?api=1&origin=foo&destination=bar` - make sure that's satisfied 😄

Reply:
Let us skip this for now, I have created a new issue to address all similar issues in this file. Commented [here](https://github.com/vishakha-lall/MapBot/pull/42#discussion_r387172016), issue #44

Reply:
@vishakha-lall - I was just thinking about this 😄